### PR TITLE
i18n: New translation - Swedish

### PR DESCRIPTION
**Pull request by sebdanielsson**
Adds Swedish translations.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,12 +2,10 @@
 
 Thank you for your interest in contributing! Join the [**Matterpoll channel**](https://community.mattermost.com/core/channels/matterpoll) on the Mattermost community server for discussion about this plugin.
 
-
```
Comment (hanzei) [inline CONTRIBUTING.md:5]: Let's add these back

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -30,6 +28,7 @@ The localization process is defined below:
 - Russian: [@ChiefAbbot](https://github.com/ChiefAbbot)
 - Simplified Chinese: [@sharuru](https://github.com/sharuru)
 - Spanish: [@ernestoittig](https://github.com/ernestoittig)
+- Swedish: [@sebdanielsson](https://github.com/sebdanielsson)
```
Comment (hanzei) [inline CONTRIBUTING.md:31]: :tada:

**Review by hanzei — COMMENTED** [review]
Thanks for volunteering to maintain the swedish translation :tada:

File: CONTRIBUTING.md (hunk 1 above)
Comment (sebdanielsson) [reply CONTRIBUTING.md:5]: Sure? They won't add more space when in preview mode and triggered some Markdown linting warnings :)
<https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md012---multiple-consecutive-blank-lines>

Comment (hanzei) [reply CONTRIBUTING.md:5]: 1/5 that it's easier to block md code if there are two new lines before a new section. not a blocker.

**Review by hanzei — APPROVED** [review]

**Comment by sebdanielsson** [comment]
Line breaks restored😊

**Review by kaakaa — APPROVED** [review]
Thanks @sebdanielsson !

**Comment by sebdanielsson** [comment]
> Thanks @sebdanielsson !

Thanks! When will the next release be out?

**Comment by hanzei** [comment]
Good question. @kaakaa Are there changes we would like to get into the next release?

**Comment by kaakaa** [comment]
That's a point. I don't have any strong opinion to be included in the next release. If #506 will be done soon (thanks to progress it @hanzei ), then I'll work to create new release.

**Pull request merged**